app/vmctl/README.md

```diff
@@ -765,6 +765,40 @@ Requests to make: 9 / 9 [██████████████████
   requests retries: 0;
 2023/03/02 09:22:06 Total time: 3.633127625s
 ```
+In case when retries with backoff policy is unneeded `--vm-native-disable-retries` command line flag can be used.
```

Comment:
Could you describe cases when disabling retries might help?
It is better to describe such cases so that users will understand benefits and penalties of using this flag.

Reply:
```suggestion
The retries policy is designed for big datasets and may have significant time- and network-resource-overhead for small datasets (e.g. a few MB) which comprise many metrics (e.g. more than hundred). In such cases `--vm-native-disable-retries` command line flag can be used.
```

Reply:
added information about benefits and penalties of the retries policy